toolchain/check/testdata/class/fail_base_bad_type.carbon

```diff
@@ -99,6 +99,10 @@ fn ConvertToBadBaseStruct(p: DeriveFromStruct*) -> {.a: i32, .b: i32}* { return
 // It would be OK to reject this if we start actually looking in the struct type.
 fn AccessMemberWithInvalidBaseStruct(p: DeriveFromStruct*) -> i32 { return (*p).n; }
 
+// CHECK:STDERR: fail_base_bad_type.carbon:[[@LINE+4]]:1: error: `base` not allowed on `class` declaration, only definition
+// CHECK:STDERR: base class Incomplete;
+// CHECK:STDERR: ^~~~
+// CHECK:STDERR:
```

Comment:
Again a case where the tests should be in separate splits. Many of these files predate us supporting splits. This is also a case where "`base` only allowed on definitions" should be separated from other tests about incomplete types.

Reply:
Made some splits here too.